homeassistant/components/droplet/config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Config flow for Droplet integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pydroplet.droplet import DropletDiscovery
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.service_info.zeroconf import ZeroconfServiceInfo
+
+from .const import (
+    CONF_DEVICE_ID,
+    CONF_DEVICE_NAME,
+    CONF_HOST,
+    CONF_MANUFACTURER,
+    CONF_MODEL,
+    CONF_PAIRING_CODE,
+    CONF_PORT,
+    CONF_SERIAL,
+    CONF_SW,
+    DEVICE_NAME,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class FlowHandler(ConfigFlow, domain=DOMAIN):
```

Comment:
```suggestion
class DropletConfigFlow(ConfigFlow, domain=DOMAIN):
```
let's give it a custom name